assembl/templates/base.jinja2

```diff
@@ -6,20 +6,46 @@
     <meta name="viewport" content="width=device-width, initial-scale = 1,user-scalable=no,maximum-scale=1.0">
 
-    {% block meta_tags %}
-    <meta name="description" content="">
-    <meta property="og:title" content="Assembl" />
-    <meta property="og:type" content="website" />
-    <meta property="og:url" content="{{ application_url }}" />
-    <meta property="og:image" content="{{ application_url }}/static/css/themes/default/img/crowd2.jpg" />
-    <meta property="og:image:width" content="1600" />
-    <meta property="og:image:height" content="222" />
-    <meta property="og:image" content="{{ application_url }}/static/css/themes/default/img/logo.png" />
-    <meta property="og:image:width" content="111" />
-    <meta property="og:image:height" content="35" />
-    <meta name="robots" content="noindex" />
-    <meta name="googlebot" content="noindex" />
-    {% endblock meta_tags %}
+    {% block meta_tags_public %}
+        {% if not private_social_sharing %}
+            <meta name="description" content="{{get_description}}">
+            <meta property="og:title" content="{{get_topic}}" />
+            <meta property="og:type" content="website" />
+            <meta property="og:url" content="{{ application_url }}" />
+            <meta property="og:image" content="{{get_landing_page_image}}" />
+            <meta property="og:image:width" content="1600" />
+            <meta property="og:image:height" content="222" />
+            <meta property="og:locale" content="{{opengraph_locale}}" />
+            <meta name="robots" content="noindex" />
+            <meta name="googlebot" content="noindex" />
+
+            <meta name="twitter:card" content="summary_large_image">
+            <meta name="twitter:site" content="@bluenove">
+            <meta name="twitter:creator" content="@bluenove">
+            <meta name="twitter:title" content="Assembl">
+            <meta name="twitter:description" content="{{get_description}}">
+            <meta name="twitter:image" content="{{get_landing_page_image}}">
+        
+        {% else %}
+
+            <meta name="description" content="">
+            <meta name="og:title" content="{{get_topic}}" />
+            <meta property="og:type" content="website" />
+            <meta property="og:url" content="{{ application_url }}" />
+            <meta property="og:image" content="{{ application_url }}/static/css/themes/default/img/crowd2.jpg" />
+            <meta property="og:image:width" content="1600" />
+            <meta property="og:image:height" content="222" />
+            <meta property="og:locale" content="{{opengraph_locale}}" />
+
+            <meta name="twitter:card" content="summary_large_image">
+            <meta name="twitter:site" content="@bluenove">
+            <meta name="twitter:creator" content="@bluenove">
+            <meta name="twitter:title" content="Assembl">
+            <meta name="twitter:description" content="">
```

Comment:
probably use {{get_description}}

Reply:
actually no ... the purpose is to make the description not visible when it is a private debate, so after discussions with fred and frank, they told me to leave this empty

Reply:
So for a private debate, please remove the meta tag if it is not a required tag. Semantically, it means something to have the meta tag with `content=""`.